sql/install.sql

```diff
@@ -3,93 +3,6 @@ SET client_min_messages TO WARNING;
 -- Create the PL/Python3 extension.
 CREATE EXTENSION IF NOT EXISTS plpython3u;
 
----
```

Comment:
This broke the Docker container from starting up correctly as the demo. This script installs everything into the `root` database, so maybe we can keep it around until we fully migrate to Django?